core/src/main/res/values/strings.xml

```diff
@@ -843,4 +839,12 @@
     <string name="on_demand_config_setting_changed">Setting updated successfully.</string>
     <string name="on_demand_config_stream_text">Looks like you stream a lot. Do you want episode lists to show stream buttons?</string>
     <string name="on_demand_config_download_text">Looks like you download a lot. Do you want episode lists to show download buttons?</string>
+    <string name="synchronization">Synchronization</string>
+    <string name="preference_synchronization_summary_unchoosen">You can choose from multiple providers to synchronize your subscriptions and episode play state with</string>
+    <string name="preference_synchronization_summary_nextcloud">Your subscriptions and episode actions are synchronized via the gpoddersync app on a nextcloud server</string>
+    <string name="synchronization_choose_and_login_summary">Choose a sync provider and login</string>
+    <string name="synchronization_choose_and_login_title">Choose sync provider</string>
+    <string name="pref_synchronization_logout_summary">You can choose your sync provider again once you have logged out</string>
+    <string name="pref_synchronization_force_sync_summary">Sync all subscriptions and episode states</string>
```

Comment:
I think force-sync should sound a bit more scary, so users are not tempted to press this button every time :) Maybe something like "Re-synchronize all subscriptions and episode states"?

Reply:
Why should users be discouraged to force sync? (It's on their own infrastructure, so it's not hurting a central service. And I'm not sure where this option is shown - presumably not in a very easily accessible location?

In a general note, I would prefer to use 'synchronisation' rather than 'sync' (not have colloquial abbreviations).

Reply:
The string is shown for both sync services. Gpodder.net has trouble with their EpisodeActions database table getting larger and larger. That button sends all episode actions again, so it does cause quite some load on their server. This button should only be pressed when actually necessary, not just for a normal, manual sync.

Reply:
Aha. Then it'd be a good change indeed.